keyboards/keebit/coral/info.json

```diff
@@ -0,0 +1,149 @@
+{
+    "manufacturer": "keeb.it!",
+    "keyboard_name": "coral",
+    "maintainer": "numToStr",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "lto": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "rows": ["GP29", "GP28", "GP14", "GP6", "GP7"],
+        "cols": ["GP1", "GP0", "GP27", "GP26", "GP15", "GP5"]
+    },
+    "processor": "RP2040",
+    "url": "https://github.com/keeb-it/coral",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP16"
+    },
+    "rgblight": {
+        "split": true,
+        "led_count": 1,
+        "hue_steps": 2,
+        "saturation_steps": 2,
+        "brightness_steps": 2,
+        "max_brightness": 20,
+        "sleep": true,
+        "layers": {
+            "blink": true,
+            "enabled": true,
+            "override_rgb": true
+        },
+        "animations": {
+            "breathing": true,
+            "christmas": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "twinkle": true
+        }
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [{ "pin_a": "GP9", "pin_b": "GP8", "resolution": 1 }]
```

Comment:
Default for the resolution is 4, which is 4 pulses between each detent.  Which is the typical amount. 

Does the board come with an encoder, and are you sure that 1 is the correct value?

Reply:
Yup, the board does come with an encoder but i never knew 1 was wrong. Now i changed it 2 which works better for the encoder that i am using.